Header header
string context_id
mirador_driver/StrategicPoint[] strategic_points